external/ag-website-shared/src/components/features-section/DocsFeaturesSection.tsx

```diff
@@ -0,0 +1,103 @@
+import type { FunctionComponent } from 'react';
+
+import styles from './features.module.scss';
+
+interface Props {
+    type: string;
```

Comment:
This can be more restrictive

```suggestion
    type: 'community' | 'enterprise';
```